docs/operations/clean-metadata-store.md

```diff
@@ -119,7 +145,23 @@ Indexer task log cleanup on the Overlord uses the following configuration:
 
 For more detail, see [Task logging](../configuration/index.md#task-logging).
 
-## Automated metadata cleanup example configuration
+
+## Disable automated metadata cleanup
+
+Druid enables automated metadata cleanup by default. To disable this feature, set the following properties in the `coordinator/runtime.properties` file:
+
+```
+druid.coordinator.kill.on=false
```

Comment:
Might be good to add a comment beside each one indicating which table/entity is the config for